src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -245,7 +295,7 @@ private bool Vote(ApplicationEngine engine, UInt160 account, ECPoint voteTo)
         [ContractMethod(1_00000000, CallFlags.AllowStates)]
         public ECPoint[] GetCommittee(StoreView snapshot)
         {
-            return GetCommitteeFromCache(snapshot).OrderBy(p => p).ToArray();
+            return GetCommitteeFromCache(snapshot).Select(p => p.PublicKey).OrderBy(p => p).ToArray();
```

Comment:
Maybe we can store them ordered by PubKey instead of by votes.

Reply:
No, we need the order in https://github.com/neo-project/neo/pull/1848/files#diff-47aeabb486b6d5eb2c7296628377ab925b88ae9d7d59a973dd2d795aab8a9acbR157

Reply:
But what should happen more, reward is not for every blocks.

Reply:
> But what should happen more, reward is not for every blocks.

What does it mean?